docs/flink_job_cluster_guide.md

```diff
@@ -0,0 +1,95 @@
+# Running Flink-Job-Cluster in Kubernetes
+
+## Overview
+
+to run a Flink-Job-Cluster, you first have to deploy the Flink-Operator. For that, follow the [user_guide](https://github.com/GoogleCloudPlatform/flink-on-k8s-operator/blob/master/docs/user_guide.md)
+
+## Prerequisites
```

Comment:
Make prerequisite as a list.
- Operator image version, deploying the operator 
- Flink image version.
- Helm version.